history/cultures/high_elf.txt

```diff
@@ -54,4 +54,5 @@
 	#
 	discover_innovation = innovation_guilds 
 	#
+	discover_innovation = innovation_kalimdor_settling
```

Comment:
Huh? What is the idea behind it?

Reply:
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/84#discussion_r606298363

This